prow/cmd/deck/main.go

```diff
@@ -314,7 +314,8 @@ func main() {
 		indexHandler := handleSimpleTemplate(o, cfg, "index.html", struct {
 			SpyglassEnabled bool
 			ReRunCreatesJob bool
-		}{o.spyglass, o.rerunCreatesJob})
+			AllowAnyone     bool
+		}{o.spyglass, o.rerunCreatesJob, cfg().Deck.RerunAuthConfig.AllowAnyone})
```

Comment:
Personally I think it is always better the do `{SpyglassEnabled: o.spyglass` as its more readable

Reply:
IntelliJ fills it in for you! But I changed it for vim users :)